german/Einladen/einladen.md

```diff
@@ -1,4 +1,4 @@
-Sie haben den Mut und naehern sich ihr an. Sie fragen sie, ob es m�glich ist, zusammen ein kaltes Getraenk zu trinken. 
+Sie nehmen Ihren ganzen Mit zusammen und naehern sich ihr an. Sie fragen sie, ob es m�glich ist, zusammen ein kaltes Getraenk zu trinken. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.